django/db/models/base.py

```diff
@@ -834,6 +834,13 @@ def save_base(self, raw=False, force_insert=False,
         # Once saved, this is no longer a to-be-added instance.
         self._state.adding = False
 
+        # Readonly fields are computed by the Database. If there are any,
+        # we should fetch their value.
+        if self._meta.has_readonly_fields:
```

Comment:
I think it would be good if there was a way to disable this extra query, e.g. in an argument to `save`. For example, if a model has a generated/computed field used purely for indexing on the database level, there's no need to fetch that field back upon `save()`